### PR TITLE
Laravel 11.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 11.x.

**Before merging**, you need to:

- Checkout the `l11-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:information_source: Shift detected you do not have a CI workflow using GitHub Actions. If you would like, Shift can help you get started with the [CI Generator](https://laravelshift.com/ci-generator). Although the Shift is paid, GitHub Actions is free and having a CI workflow will improve the stability and review process for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/cas.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "subfission/cas": "dev-l11-compatibility",
    }
}
```

Finally, run: `composer update`

File: composer.json [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
-  },
-  "require-dev": {
-    "phpunit/phpunit": "^8.0|^9.0|^10.0",
-    "pestphp/pest": "^1.21"
-  },
-  "autoload": {
-    "psr-4": {
-      "Subfission\\Cas\\": "src/Subfission/Cas/"
+    "name": "subfission/cas",
+    "description": "Adds CAS support to Laravel 5.x - 10.x",
```
Comment (jcottrell) [inline composer.json:3]: to 11.x though, right?

Comment (subfission) [reply composer.json:3]: Yes.  The bot wants require to also include a reference to 11.

**Pull request closed without merging**